wandb/sdk/wandb_settings.py

```diff
@@ -16,13 +16,17 @@
 
 Priority of settings:  See "source" variable.
 
+When override is used, it has priority over non-override settings
+
+Override priorities are in the reverse order of non-override settings
```

Comment:
I guess this is the confusing part.  Is there another word that could better describe this?

Reply:
Maybe `_priority="default"` and `_priority="reverse"`.  Is there ever a usecase for `_priority="force"`?

Reply:
i think "force" is what override is, i can write some more examples...